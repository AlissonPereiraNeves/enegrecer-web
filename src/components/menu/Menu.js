import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import logo from '../../imagens/logo.svg';
import './menu.css';

export default class Menu extends Component {
  componentWillMount() {
  }

  render() {
    return (
      <div className="menu">
        <div className="logo">
          <Link to="/">
            <img src={logo} alt="Logo Verdade Seja Dita" />
          </Link>
        </div>
        <div className="links">
          <Link to="/sobre">SOBRE</Link>
          <Link to="/depoisdenuncia">E DEPOIS DA DENÃšNCIA?</Link>
          <Link to="/coletivo-enegrecer">O COLETIVO ENEGRECER</Link>
          <Link to="/painel/denuncias/nova">DENUNCIAR</Link>
        </div>
      </div>
    );
  }
}
