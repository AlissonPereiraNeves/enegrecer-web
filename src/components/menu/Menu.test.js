import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Menu from './Menu';

describe('Menu', () => {
  it('deve construir a pagina corretamente ', () => {
    const menu = shallow(<Menu />);
    const menuJson = toJson(menu);
    expect(menuJson).toMatchSnapshot();
  });
});
