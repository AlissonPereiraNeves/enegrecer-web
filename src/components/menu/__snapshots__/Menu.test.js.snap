// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu deve construir a pagina corretamente  1`] = `
<div
  className="col s12 m6 menu"
>
  <div
    className="logo"
  >
    <Link
      replace={false}
      to="/"
    >
      <img
        alt="Logo Verdade Seja Dita"
        src="logo.svg"
      />
    </Link>
  </div>
  <div
    className="mobile_nav"
    key="7"
  >
    <a
      onClick={[Function]}
      role="button"
      tabIndex="0"
    >
      <i
        className="material-icons icon-white"
      >
        menu
      </i>
    </a>
  </div>
</div>
`;
